src/OrchardCore.Modules/OrchardCore.Flows/ViewModels/BagPartViewModel.cs

```diff
@@ -6,6 +8,7 @@ namespace OrchardCore.Flows.ViewModels
     public class BagPartViewModel
     {
         public BagPart BagPart { get; set; }
+        public List<ContentItem> ContentItems { get; set; }
```

Comment:
Why are you adding this? We can already access them from the data model, or from the `BagPart` property.

Reply:
E.g you can access `Model.ContentItem.Content.TitlePart.Title` because the model type is register and also the `ContentItem` type. Then because `Content` is a JObject you can access sub properties.

But if you override a `BagPart` template, you can access `Model.BagPart` **but, because `BagPart` is not a registered type, you can't access `Model.BagPart.ContentItems`**.

Maybe more modules would need to register some of their types. **One suggestion** would be that when we register a type, **we could also register the types of their properties** which are not a simple type, at least at one more level.

Reply:
Can you try with just `List<ContentItem> ContentItems => BagPart.ContentItems` then ?

Reply:
And remove the assignment from the driver

Reply:
Okay